src/test/TestResult.java

```diff
@@ -5,8 +5,8 @@
 public class TestResult {
 
     public static void main(String[] args) {
+	int[] arr = {10,6,4,3,2,3,0};
+        System.out.printf("the number that repeats the largest number of times: %d",Exercise.Calculate(arr));      

```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.